tests/gateways/test_repodata_gateway.py

```diff
@@ -319,14 +329,14 @@ def test_cache_json(tmp_path: Path):
     state = RepodataState(cache_json, cache_state, "repodata.json")
     state.mod = mod  # this is the last-modified header not mtime_ns
     state.cache_control = "cache control"
-    state.etag = "etag"
+    state.etag = '"unambiguous-etag"'
```

Comment:
whats the purpose of having the double quotes here?

Reply:
Etags have quotes in them normally